docs/src/main/asciidoc/security-openid-connect-client-reference.adoc

```diff
@@ -1117,6 +1117,62 @@ The `quarkus-oidc-token-propagation-reactive` extension provides `io.quarkus.oid
 The `quarkus-oidc-token-propagation-reactive` extension (as opposed to the non-reactive `quarkus-oidc-token-propagation` extension) does not currently support the exchanging or resigning the tokens before the propagation.
 However, these features may be added in the future.
 
+[[oidc-client-graphql-client]]
+== GraphQL client integration
+
+The `quarkus-oidc-client-graphql` extension provides a way to integrate an
+OIDC client into xref:smallrye-graphql-client.adoc[GraphQL clients]. This
+works similarly as with REST clients. When this extension is present, any
+configured (that means NOT created programmatically via the builder, but via
+configuration properties) GraphQL client will attempt to use the OIDC client
+to obtain an access token and set it as an `Authorization` header value.
+OIDC client will also refresh expired access tokens.
+
+To configure which OIDC client should be used by GraphQL client, select one of the configured OIDC clients with the `quarkus.oidc-client-graphql.client-name` property, for example:
+
+----
+quarkus.oidc-client-graphql.client-name=oidc-client-for-graphql
+
+# example declaration of the OIDC client itself
+quarkus.oidc-client.oidc-client-for-graphql.auth-server-url=http://example.com/realms/quarkus
+quarkus.oidc-client.annotation.grant-options.password.username=${username}
```

Comment:
```suggestion
quarkus.oidc-client.oidc-client-for-graphql.grant-options.password.username=${username}
```

Reply:
Please make this OIDC client config complete, (client id and secret properties for this test client, setting grant type to password), users sometimes copy and paste examples and then they might wonder why it does not work for the password (I've definitely had such queries before, quite likely due to incompleted fragments in this guide :-) )

Reply:
Done, please check

Reply:
Thanks @jmartisk